quetz/migrations/versions/546854af251f_add_channel_metadata.py

```diff
@@ -0,0 +1,71 @@
+"""add channel metadata
+
+Revision ID: 546854af251f
+Revises: ea6eba9a9ffc
+Create Date: 2021-02-09 16:41:09.831657
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '546854af251f'
+down_revision = 'ea6eba9a9ffc'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('channels', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('channel_metadata', sa.String(), nullable=True))
+        batch_op.alter_column(
```

Comment:
why is this generated? did you change the type of size and limit_size?

Reply:
I just added a new field, rest of stuff is auto-generated by alembic.
Ideally this was covered in a previous migration when we decided to use BigIntegers for sizes.
So maybe we can manually remove this?

Reply:
Fixed in latest commit